Update the Community landing page (#717)
neuroinformatics-unit/movement: docs/source/snippets/connect-with-us.md

|  | Platform | Come here to |
|---|----------|---------------|
| {fas}`comments` | [Zulip](movement-zulip:) | Ask general questions, seek user support, chat with `movement` users and developers. |
| {fab}`github` | [GitHub](movement-github:) | [Open an issue](https://github.com/neuroinformatics-unit/movement/issues) to report a bug or request a new feature. Open a pull request to contribute code or documentation (see [contributing guide](target-contributing)). Star the [repository](movement-github:) if you like `movement`. |
| {fas}`video` | Community Calls | Meet the team and chat about any aspect of `movement` development. [Follow this Zulip topic](movement-community-calls:) to receive updates about upcoming calls. These typically run every other Friday from 11:00 to 11:45 (London, U.K. time). |

> | {fab}`bluesky` {fab}`mastodon` | Social Media | Follow project announcements, opportunities and upcoming events, on [Bluesky](https://bsky.app/profile/neuroinformatics.dev) and [Mastodon](https://mastodon.online/@neuroinformatics). |

Reviewer: ```suggestion
| {fab}`bluesky` {fab}`mastodon` | Social Media | Catch the latest project announcements, opportunities and upcoming events over on [Bluesky](https://bsky.app/profile/neuroinformatics.dev) and [Mastodon](https://mastodon.online/@neuroinformatics). |
```
or 
Follow us on Bluesky and Mastodon for project announcements, opportunities and upcoming events

Reviewer: Or change "Social Media" to links to bluesky and mastodon, and leave these out of the "Come here to" column
```suggestion
| {fab}`bluesky` {fab}`mastodon` | [Bluesky](https://bsky.app/profile/neuroinformatics.dev) and [Mastodon](https://mastodon.online/@neuroinformatics) | Catch the latest project announcements, opportunities and upcoming events. |
```
